extensions/default/src/commandsModule.ts

```diff
@@ -541,6 +546,82 @@ const commandsModule = ({
         overlays.item(i).classList.toggle('hidden');
       }
     },
+
+    updateViewportDisplaySet: ({
+      direction,
+      excludeNonImageModalities,
+    }: UpdateViewportDisplaySetParams) => {
+      const nonImageModalities = [
+        'SR',
+        'SEG',
+        'SM',
+        'RTSTRUCT',
+        'RTPLAN',
+        'RTDOSE',
+      ];
+
+      const currentDisplaySets = displaySetService.activeDisplaySets;
+
+      // Sort the display sets as per the thumbnail/study browser list
+      utils.sortBySeriesDate(currentDisplaySets);
```

Comment:
so isn't the currentDisplaySets sorted by default? I'm confused why we need to sort again.

Reply:
I found that is was NOT sorted. Without this line, the navigation would jump to display sets in an order different than what appears in the thumbnail list.